src/main/java/com/flightstats/hub/dao/aws/DynamoWebhookDao.java

```diff
@@ -74,13 +76,23 @@ public Webhook get(String name) {
             if (result.getItem() == null) {
                 return null;
             }
-            return mapItem(result.getItem());
+            return tryMapItem(result.getItem())
+                    .orElseThrow(() -> new ResourceNotFoundException("Unable to read webhook config from dynamo " + name));
         } catch (ResourceNotFoundException e) {
             log.warn("group not found {}", name, e);
             return null;
         }
     }
 
+    private Optional<Webhook> tryMapItem(Map<String, AttributeValue> item) {
+        try {
+            return Optional.of(mapItem(item));
+        } catch (Exception e) {
+            log.warn("Unable to map webhook config {} {}", item.get("key"), e.getMessage());
```

Comment:
It's generally an anti-pattern to eat the underlying exception as that loses the stack trace. Usually the log line is:
`log.warn("bad things happened", e)`
Unless there's a specific reason not to? Like maybe this throws exceptions a LOT and it'd flood the logs?

Reply:
Aha. I copy-pasta'd this line from the channel config DAO. I'll fix it in both places.

Reply:
Woohoo, my review found something useful!

Reply:
You say this like it's not a common occurrence. ;)